core-services/prow/02_config/_plugins.yaml

```diff
@@ -1785,6 +1785,9 @@ override:
   allow_top_level_owners: true
 owners:
   filenames:
+    openshift-kni/scheduler-plugins:
+      owners: DOWNSTREAM_OWNERS
```

Comment:
To what OWNERS file in the ```openshift-kni/scheduler-plugins``` this ```DOWNSTREAM_OWNERS``` are mapped to?

Reply:
here: https://github.com/openshift-kni/scheduler-plugins/pull/19

Reply:
I think it is better to have the file first.
I don't know the outcome of the missing file for prow.

Reply:
you just did so it's great